docs/src/main/paradox/jms.md

```diff
@@ -40,256 +40,236 @@ Gradle
 
 ## Usage
 
-The JMS message model supports several types of message body (see @javadoc[javax.jms.Message](javax.jms.Message)) and Alpakka currently supports messages with a body containing a @javadoc[String](java.lang.String) object.
+The JMS message model supports several types of message body (see @extref(javaee-api:javax.jms.Message)) and Alpakka currently supports messages with a body containing a @extref[String](java-api:java.lang.String), 
+a @extref[Serializable](java-api:java.io.Serializable) object, a @extref[Map](java-api:java.util.Map) or a byte array.
 
-Use the case class @scaladoc[JmsTextMessage](akka.stream.alpakka.jms.JmsTextMessage$) to wrap the messages you want to send and optionally set their properties (see below for an example).
+Use a case class with the subtype of @scaladoc[JmsMessage](akka.stream.alpakka.jms.JmsMessage$) to wrap the messages you want to send and optionally set their properties (see below for an example).
 
 ### Sending messages to a JMS provider
```

Comment:
Change to `##` levels for the sections show in the left hand overview.

Reply:
Documentation is restructured. Please have a look